docs/sources/enterprise/query-caching.md

```diff
@@ -21,20 +21,20 @@ The following cache backends are available: in-memory, Redis, and Memcached.
 - Reduced API costs.
 - Reduced likelihood that APIs will rate-limit or throttle requests.
 
-## Enable query caching
+## Enable and configure query caching
 
-> **Note:** The query caching feature is enabled by default. To start caching data source queries, visit the data source's configuration page.
+By default, data source queries are not cached. To enable query caching for a single data source, visit the data source settings page, open the Cache tab, and press the Enable button. Once enabled, you can choose a custom TTL for that data source or keep the default TTL.
 
-To enable and configure query caching, please refer the the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#caching" >}}).
+> **Note:** Not all data sources work with query caching. If the Cache tab is not visible in a data source's settings, then the data source does not support query caching.
```

Comment:
Try to avoid possessive with inanimate objects such as data sources. It's just weird, and potentially confusing for non-native English readers.

Reply:
Fair enough :) I gave it a try with a rewording.